docs/backends/index.md

```diff
@@ -1,51 +1,37 @@
 # Backends
 
-This document describes the classes of backends, how they work, and any details
-about each backend that are relevant to end users.
-
-For more information on a specific backend, check the next backend pages:
-
-There are currently three classes of backends that live in ibis.
-
-1. String generating backends
-1. Expression generating backends
-1. Direct execution backends
-
 ## String Generating Backends
 
-The first category of backend translates ibis expressions into strings.
-Generally speaking these backends also need to handle their own execution. They
-work by translating each node into a string, and passing the generated string
-to the database through a driver API.
+The first category of backend translate Ibis expressions into strings.
+Generally speaking these backends handle their own execution.
+
+The compiler turns each expression into a string and passes that string to the
+database through a driver API.
 
-- [Apache Impala](https://impala.apache.org/)
-- [Clickhouse](https://clickhouse.yandex/)
-- [Google BigQuery](https://cloud.google.com/bigquery/)
-- [Hadoop Distributed File System (HDFS)](https://hadoop.apache.org/)
-- [OmniSciDB](https://www.omnisci.com/)
+- [Apache Impala](Impala.md)
+- [ClickHouse](ClickHouse.md)
+- [Google BigQuery](https://github.com/ibis-project/ibis-bigquery/)
+- [OmniSciDB](https://github.com/omnisci/ibis-omniscidb)
 
 ## Expression Generating Backends
 
-The second category of backends translates ibis expressions into other
-expressions. Currently, all expression generating backends generate [SQLAlchemy
-expressions](http://docs.sqlalchemy.org/en/latest/core/tutorial.html).
+The next category of backends translates ibis expressions into other
```

Comment:
```suggestion
The next category of backends translates ibis expressions into other expression based representations. 
```

Reply:
🤷🏻 I could go either way on this. I'm not sure what additional information is conveyed by `expression based representations` versus `expressions`.